components/DatePicker/AvailabilityDatePicker.js

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect } from "react";
+import DatePicker from "react-datepicker";
+import { addMonths, endOfMonth } from "date-fns";
+
+import "react-datepicker/dist/react-datepicker.css";
+
+export function AvailabilityDatePicker({
+  onDateChange,
+  onShiftChange,
+  excludeDates,
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedDate, setSelectedDate] = useState(new Date());
+  const [selectedShift, setSelectedShift] = useState("");
+
+  const minDate = new Date();
+  const maxDate = endOfMonth(addMonths(new Date(), 1));
+
+  function handleClick(e) {
+    e.preventDefault();
+    setIsOpen(!isOpen);
+  }
```

Comment:
Since the event is not a submit event, you don't need to prevent the default behavior.